Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/jamulus/1-index/horizontal-auto.svg)

pljones commented:
Is it possible to put a lock on Weblate for translations until #990 is also merged?

ignotus666 commented:
Yes, but even if more translations are done in the meantime it's simply a matter of making sure to commit and push them right before merging #990. I'd like to review the translations first though to see if anything obvious needs correcting.

ann0see commented on _translator-files/po/de/1-index.po:
```diff
@@ -106,19 +110,19 @@ msgstr "Willst du mithelfen?"
 #, no-wrap
 msgid "Ideas? Found a bug? Want to contribute some code or help [translating](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md) Jamulus into your language? Since Jamulus is [free and open source software](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) licensed under the [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) you can help us!"
-msgstr ""
+msgstr "Ideen? Einen Fehler gefunden? Willst Du ein bisschen Code beitragen, oder helfen Jamulus in Deine Sprache zu [übersetzen](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md)? Da Jamulus eine [freie und Open-Source-Software](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) ist und unter der [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.htm) lizenzie ...
```
No. This switches between Du and Sie. This should all be changed to "du".

ann0see commented on _translator-files/po/de/1-index.po:
```diff
@@ -106,19 +110,19 @@ msgstr "Willst du mithelfen?"
 #, no-wrap
 msgid "Take a look at our [contribution guidelines](wiki/Contribution) to find out how. Everybody is welcome!"
-msgstr ""
+msgstr "Schau in unsere [Beitragsrichtlinien](wiki/Contribution), um herauszufinden, wie es funktioniert. Jeder ist willkommen!"
```
how it works (wie es funktioniert) should be rephrased.

ann0see commented on _translator-files/po/de/1-index.po:
```diff
@@ -106,19 +110,19 @@ msgstr "Willst du mithelfen?"
 #, no-wrap
 msgid "_For detailed information on how Jamulus works, see [this paper by Volker Fischer (PDF)](/PerformingBandRehearsalsontheInternetWithJamulus.pdf)._\n"
-msgstr ""
+msgstr "_Für detaillierte Informationen, wie Jamulus funktioniert, schaue in [dieses Paper von Volker Fischer (PDF)](/PerformingBandRehearsalsontheInternetWithJamulus.pdf)._\n"
```
Very bulky.

pljones commented:
Can we get the Weblate automation to put new PRs into the "Tracking" project?  Alternatively, that could be done manually when setting the target release.  I'm assuming this is for 3.11.0?

ann0see commented:
@ignotus666 @trebmuh @jujudusud @BLumia could you please check this for obvious language mistakes?

ann0see commented:
Not sure who is currently responsible for pt_\*

BLumia commented on _translator-files/po/zh_CN/1-index.po:
```diff
@@ -57,21 +42,22 @@ msgid ""
 msgstr ""
 "---\n"
-"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
+"title: \"Jamulus - 与朋友一起免费的在线音乐播放。\"\n"
```
```suggestion
"title: \"Jamulus - 与朋友一起免费的在线演奏音乐。\"\n"
```

The term "Play" should be 演奏 instead of 播放. Other changes look fine.

I've updated the associated translation string from Weblate's side, might need to trigger an update to ensure the PR also gets updated.

ann0see replied:
Ok. Pushed the latest changes from weblate. Thanks!

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/software-manual/
Translation: Jamulus/Software-Manual

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Tips-Tricks-More
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/

ann0see commented:
Weblate is now LOCKED.
